src/views/GitHub.vue

```diff
@@ -59,3 +49,135 @@ export default {
   },
 };
 </script>
+<style>
+* {
+  font-family: "Montserrat";
+}
+
+.bg_color {
```

Comment:
use `-` in css class name and do not use `class_name` or `className`
instead use
`class-name`